### PR TITLE
Corpsman Glasses

### PR DESCRIPTION
**Pull request by Unkn0wnGh0st333**
## About the PR
Added Corpsman Glasses to the game, added it to Corpsman's lockers, and added a crafting recipe for them.

## Why / Balance
The Corpsman gets flashed at least 2 billion times per game. This is basically a SecMed hud with flash protection, something all Corpsmen dream of.

## Technical details
Added Corpsman Glasses, Replaced Security Glasses with Corpsman Glasses in Corpsman lockers, Added crafting recipe for Corpsman Glasses based off of the Security Glasses except using a MedSecHud. Added MedSecHuds to the Tags list to be used in crafting.

## Media
image
image

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
¯\_(ツ)_/¯

**Changelog**
🆑 
- Added Corpsman Glasses, no more getting flashed as Corpsman!

### CODE REVIEW

**Comment by deltanedas** [comment]
changelog is yapping make it 1 entry

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -12,3 +12,33 @@
     modifiers:
       coefficients:
         Caustic: 0.85
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
+  id: ClothingEyesGlassesCorpsman
+  name: Corpsman Glasses
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml]: corpsman glasses, lowercase

**Comment by Unkn0wnGh0st333** [comment]
> changelog is yapping make it 1 entry

my baaaad

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml [hunk 2]
```diff
@@ -12,3 +12,33 @@
     modifiers:
       coefficients:
         Caustic: 0.85
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:17]: if you are going to use contraband use base security contraband

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml [hunk 3]
```diff
@@ -12,3 +12,33 @@
...
+  id: ClothingEyesGlassesCorpsman
+  name: Corpsman Glasses
+  description: Security glasses designed for the Corpsman for medical and security needs. Now with a cool blue hue*
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:20]: `*`

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml [hunk 4]
```diff
@@ -12,3 +12,33 @@
...
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/corpsglasses.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml]: move the sprite to DeltaV folder and update everything

File: Resources/Prototypes/Recipes/Construction/clothing.yml [hunk 5]
```diff
@@ -108,6 +108,17 @@
   icon: { sprite: Clothing/Eyes/Glasses/secglasses.rsi, state: icon }
   objectType: Item
 
+- type: construction
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Construction/clothing.yml]: deltav folder

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+- type: constructionGraph
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml]: deltav folder

File: Resources/Prototypes/tags.yml [hunk 7]
```diff
@@ -674,6 +674,9 @@
 - type: Tag
   id: HudSecurity
 
+- type: Tag
```
Comment (deltanedas) [inline Resources/Prototypes/tags.yml]: deltav folder

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml (hunk 2 above)
Comment (Unkn0wnGh0st333) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:17]: image
The normal security glasses also use the "Restricted Contraband"

Comment (Unkn0wnGh0st333) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:17]: there is also no "BaseSecurityContraband"

**Review by deltanedas — DISMISSED** [review]

**Comment by Unkn0wnGh0st333** [comment]
woops forgot one thing

**Review by deltanedas — APPROVED** [review]

**Pull request merged**